msupply-foundation/open-msupply
# 2953 supplier return list view

Fixes #2953

# 👩🏻‍💻 What does this PR do? 
 

![Screenshot 2024-02-16 at 2 57 54 PM](https://github.com/msupply-foundation/open-msupply/assets/55115239/3df5587f-3cb8-4c9c-beb9-007681667791)

# 🧪 How has/should this change been tested? 


- Go to Replenishment > Outbound Returns
- See table view
- Selecting lines + delete calls the mock delete mutation
- Export to CSV works
- Create return button opens the supplier name modal, but doesn't go any further after that yet :)

Note: I think the filters work, but we'll know once the API actually returns data based on what we pass in 😁 

## 💌 Any notes for the reviewer?


Some code is commented out for now, but left for reference: handling the create button, disabling verified rows...

Probably going to be some merge conflicts with this/Carl's branch/my others... but didn't want to keep using previous as a base and creating a dependency on what PRs can be merged in first... feel like I have a pretty good sense of where the overlap will be and should be manageable though :)

...Also super indecisive PR on the outbound/supplier return naming front... started on the frontend with outbound return, but then supplier return was already in the backend so now there's a bit of both 🙈  I'll update this PR to whichever we choose when we do!
